### DISCUSSION TITLE
유니티에서 'playground' 파일 열 때 에러

### DISCUSSION THREAD

**Discussion by ahnhayden46** [naverz/zepeto-studio-kor#790] [5. Q&A]
안녕하세요, 첫 3d 아이템을 만들어보려고 하는데 유니티에서 자꾸 에러가 나 질문드립니다.

'zepeto-studio-unity-3.1.11.1' 프로젝트 폴더 안에서 'playground' 파일을 열려고 할 때 'Resolving Packages'에서 막히는 듯 했다가 곧이어 밑에 메시지가 뜹니다.

 COMMAND LINE ARGUMENTS:
D:\Utilities\Unity\2020.3.9f1\Editor\Unity.exe
-openfile
D:\Zepeto\zepeto-studio-unity-3.1.11.1\Assets\Playground.unity
Opening project via file: D:/Zepeto/zepeto-studio-unity-3.1.11.1/Assets/Playground.unity
D:/Zepeto/zepeto-studio-unity-3.1.11.1
Using Asset Import Pipeline V2.
[Package Manager] Done resolving packages in 121.41s seconds
An error occurred while resolving packages:
  Package zepeto.character@3.1.11 has invalid dependencies or related test packages:
    zepeto.asset (dependency): Request [GET https://upm.zepeto.run/zepeto.asset] failed with status code [504]
  One or more packages could not be added to the local file system:
    zepeto.character: Request [GET https://cdn.upm.zepeto.run/npm/gamesystem/zepeto.character/-/zepeto.character-3.1.11.tgz] failed with status code [504]

A re-import of the project may be required to fix the issue or a manual modification of D:/Zepeto/zepeto-studio-unity-3.1.11.1/Packages/manifest.json file.

메시지를 무시하면 파일을 열 수는 있지만 임포트한 블렌더 파일을 눌렀을 때 제페토 스튜디오 패키지 메뉴가 뜨질 않습니다.
유니티 재설치도 해보고 파일을 다시 받기도 해봤는데 해결이 되지 않았습니다.
도와주실 수 있다면  감사하겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 패키지 관련하여 일시적인 장애가 있었고 복구 완료하였습니다.
불편 드려 죄송합니다. 
감사합니다.
